pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1555,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-syncStatusTicker:
+			glog.V(4).Infof("Periodic status sync")
+			kl.syncStatus()
+			continue
```

Comment:
One thing to note is that if we separate the loops we'll need to lock around the pods. I think we are moving that route anyways, but just a heads-up.